documentation/helpers/ArgTypes/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Tabs } from '@spark-ui/tabs'
```

Comment:
@soykje maybe you need to add dependencies in the root package.json to remove that error ? 

```
"dependencies": {
    "@spark-ui/theme-utils": "*",
    "@spark-ui/tabs": "*", // add this line
  },
```

Reply:
Oh yes... Maybe also it wasn't such a good idea to put this new `ArgTypes` in a dedicated PR, as it relies on tabs incoming component? :thinking:

Reply:
Maybe we can update this file after the Tab component is merged?

Reply:
@acd02 that's what I meant when I said I would not do a separate PR for that because we need Tabs first ahah.

What about we introduce a "WIP/Preview" section in storybook ? For components that are not ready to use ? This way we could still merge and iterate ?

We could set `Tabs` version to 0, set its meta to `Alpha/Components/Tabs` (or "Work in progress" instead of alpha) That way in SB it won't be shown with other components.

WDYT ?

Reply:
I'm not so fond about that 😅 quick question, can't we use the Tab component from Storybook?